package.json

```diff
@@ -113,7 +113,7 @@
     "@babel/parser": "7.25.6",
     "@babel/traverse": "7.25.6",
     "@babel/types": "7.25.6",
-    "@hono/vite-dev-server": "0.18.0",
+    "@hono/vite-dev-server": "^0.18.1",
```

Comment:
I fixed the version intentionally without `^` to prevent unexpected behavior (I know it's a rare case) when the user installs the version not specified. So, it should be `0.18.1`.